pegasus/sites.v3/code.org/public/css/page/csd-styles.scss

```diff
@@ -73,6 +73,11 @@
 
 .carousel-wrapper {
   padding-bottom: 0;
+
+  .carousel-wrapper p {
```

Comment:
The paragraphs are still showing the wrong font — this `.carousel-wrapper` class can be removed and then it should work 🙌